client/ayon_cinema4d/api/pipeline.py

```diff
@@ -263,3 +271,14 @@ def imprint_container(
     }
 
     lib.imprint(container, data, group="AYON")
+
+
+def on_task_changed():
+
+    if not is_headless_mode_enabled():
+        # Get AYON Context menu command plugin (menu item) by its unique id.
+        ayon_context = c4d.plugins.FindPlugin(1064309)
```

Comment:
Update to registered plugin id
```suggestion
        ayon_context = c4d.plugins.FindPlugin(1064692)
```